cmd/sops/decrypt.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+
+	"go.mozilla.org/sops"
+	"go.mozilla.org/sops/cmd/sops/codes"
+	"go.mozilla.org/sops/cmd/sops/common"
+	"go.mozilla.org/sops/keyservice"
+	"gopkg.in/urfave/cli.v1"
+)
+
+type DecryptOpts struct {
```

Comment:
I'm assuming we don't need to export those, since this is a `main` package. Keeping them private might be cleaner.